dev/README.md

```diff
@@ -0,0 +1,32 @@
+# Setting up a Development Environment
+This describes how to set up a development environment suitable for working on the automation for `gatling-puppet-load-test`
+
+## What you get
+The beaker scripts in the `beaker` directory will set up a machine with a jenkins server, a copy of this repo (which contains the gatling executables), `sbt` for running gatling, and an installation of `jenkins-job-builder` configured to point at the jenkins server.
+
+## Requirements
+This guide assumes you have a centos-6 machine/VM with:
+  * ssh key authentication for the root user (for `beaker`)
+  * `puppet` installed and in the `PATH`
+
+This will be your jenkins/gatling box 
+
+To actually run gatling against a SUT, you'll need to set up another machine with some flavor of puppetserver/PE depending on what you're trying to test. Running tests isn't covered in this guide though.
+
+It's also important to note that if you want to work on the cobbler provisioning portion of the automation, it's much more involved. There will be a separate guide for that at some point.
+
+## Setting up the jenkins/gatling box
+To run the beaker script, you must first edit `beaker/target_machine.yml` and replace `target_hostname` with the hostname or IP of your system.
```

Comment:
+1.

Also, might change `target_hostname` in that yaml file to something that might serve as a reasonable default, e.g. `jenkins-gatling`, and then suggest that devs add an entry to their /etc/hosts file (maybe suggest modifying the yml file as a backup strategy, but suggest /etc/hosts first).  This way you're not encouraging people to make user-specific changes to files that live in VCS, because once they do that they have to be careful not to commit their changes.